include/mega/logging.h

```diff
@@ -158,24 +350,64 @@ class SimpleLogger {
     template <typename T>
     SimpleLogger& operator<<(T* obj)
     {
+        const char* null = "(NULL)";
```

Comment:
could this go static into SimpleLogger?

Reply:
This is just a stack-based pointer to a static string so should be fine, no? This is essentially free code AFAIK because the compiler knows the address of `null` at compile time, which just references a string from the static storage section of the binary.

Reply:
With the introduction of `ENABLE_LOG_PERFORMANCE` it made more sense to use "(NULL)" inline again so I got rid of `null`.